[READY] RELEASE THE DRAGONS!!

Strazyplus opened the pull request:
### **[Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0)](https://creativecommons.org/licenses/by-nc-sa/3.0/)**

![1595361337 miz-dee_okay-export](https://user-images.githubusercontent.com/6706750/88109961-612c5c80-cb79-11ea-8675-99e93196e122.gif)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
Commissioned artist: deviantart.com/mizartz
Vorebellies: @ThighHighs

Though a commission with the talented Mizartz we have created a new subset of borgs for all to have, instead of being a dog the robot is a dragon or in this case a drake. the modules include Medical, Security, Mining, Peace, Engineering, and Janitor variants. in addition of  providing new borgshells we have also included plushie variants that can be obtained via plushie crate or via loadout and come in security and medical. 

![janitor](https://user-images.githubusercontent.com/6706750/88110509-56be9280-cb7a-11ea-9520-0e712ed6f325.gif)
![mining](https://user-images.githubusercontent.com/6706750/88110511-57572900-cb7a-11ea-9fab-29fee8b2f8dd.gif)
![peace](https://user-images.githubusercontent.com/6706750/88110512-57572900-cb7a-11ea-9c89-95c99773f98a.gif)
![security](https://user-images.githubusercontent.com/6706750/88110513-57572900-cb7a-11ea-8132-fa5ddfbc5c89.gif)
![engineering](https://user-images.githubusercontent.com/6706750/88110515-57efbf80-cb7a-11ea-96ad-00c43b64905a.gif)
![medic](https://user-images.githubusercontent.com/6706750/88110516-57efbf80-cb7a-11ea-8704-ca08dc83f998.gif)
![ca2rd](https://user-images.githubusercontent.com/6706750/87237091-b4572000-c3bf-11ea-904e-b42853267d4e.png)

Drakeborg and drake plushies fall under [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/)
Share — copy and redistribute the material in any medium or format, Adapt — remix, transform, and build upon the material, however NonCommercial use — You may not use the material for commercial purposes.

## Why It's Good For The Game

Brings new subset of robots for borg players as well cute plushies for people to squeeze.

## Changelog
:cl:
add: Added drake drakeborg
imageadd: added drakeborg
add: Added drake drakeplushies
Code: Added drakeplush to item code
[CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/)
add: Added CC BY-NC-SA 3.0 license details to icon/mob/cyborg
rebuild of Drakeborgs
downstream compatible with citidel
adds drakeborg
adds drakeplushies to loadout.

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Strazyplus committed:
RELEASE THE DRAGONS!!

Strazyplus commented on tgstation.dme:
```diff
@@ -3601,8 +3601,8 @@
 #include "modular_skyrat\code\game\objects\items\circuitboards\computer_circuitboards.dm"
 #include "modular_skyrat\code\game\objects\items\circuitboards\machine_circuitboards.dm"
 #include "modular_skyrat\code\game\objects\items\devices\multitool.dm"
-#include "modular_skyrat\code\game\objects\items\devices\ttsdevice.dm"
```
![ChillyEmbarrassedAnhinga-small](https://user-images.githubusercontent.com/6706750/87103909-5c37e680-c224-11ea-88e7-ccfada9e240e.gif)
why? don't ask me why github does what the fuck it wants to do.

Strazyplus commented on tgstation.dme:
```diff
@@ -3601,8 +3601,8 @@
 #include "modular_skyrat\code\game\objects\items\devices\multitool.dm"
-#include "modular_skyrat\code\game\objects\items\devices\ttsdevice.dm"
 #include "modular_skyrat\code\game\objects\items\devices\traitordevices.dm"
+#include "modular_skyrat\code\game\objects\items\devices\ttsdevice.dm"
```
![ChillyEmbarrassedAnhinga-small](https://user-images.githubusercontent.com/6706750/87104129-f730c080-c224-11ea-8ed7-87b927fe6ed5.gif)
Why? no idea. I didn't move it.

BlackMajor commented:
![Image of Derg](https://cdn.discordapp.com/attachments/161574200581029888/730949650084921455/unknown.png)

Your janitor drake bellyup sprites have the wrong eye colour and are uh.. offset eyes.

Strazyplus committed:
bellyup sprite fix + plush code tweak

Strazyplus commented:
![image](https://user-images.githubusercontent.com/6706750/87112183-323cef00-c239-11ea-98d9-6b6d77c3b294.png)
fixed

Strazyplus commented:
PR is complete, 100% bug fixing if anyone finds anything wrong.

BlackMajor commented:
I mean if you wanted to go all the way ideally you'd also have ratvar eye overlays just for consistency (rip clockwork cult)

Strazyplus commented:
> I mean if you wanted to go all the way ideally you'd also have ratvar eye overlays just for consistency (rip clockwork cult)

wait where are those sprites stored at? I can make glitchy eyes ez peezy

BlackMajor commented:
> > I mean if you wanted to go all the way ideally you'd also have ratvar eye overlays just for consistency (rip clockwork cult)
> 
> wait where are those sprites stored at? I can make glitchy eyes ez peezy

When borg's get ratvar'd they get a different eye overlay, borgname_e_r, and flavourtext
These are generally just blazing bronze/gold eye overlays.
I just used a batch filter when porting borg sprites that didn't have them, but hey, if you want to go the extra mile and have the ratvar overlays be animated, that's some work to do.

KasparoVy commented:
As mentioned on the last PR the walking animation's confusing but the sprites are bloody awesome. Like taking one step, pausing then taking another step with the same 'side' of legs, as if limping

Strazyplus commented:
> As mentioned on the last PR the walking animation's confusing but the sprites are bloody awesome. Like taking one step, pausing then taking another step with the same 'side' of legs, as if limping

borgs are slow in game. but I do see what you mean but I believe its less noticeable in-game.

KasparoVy commented:
@Strazyplus I primarily focus on the actual animations under checks before commenting so I don't end up with my foot in my mouth 😄 I understand about the GIFs.

Strazyplus committed:
Artist credit tweak

Strazyplus committed:
TIME TO TEESTMERGE DAMIT

BlackMajor commented:
Headlight overlay for drakesec and drakeeng are scuffed and backwards

Strazyplus committed:
howinthegoddamndidimissthesetwoohgodwhyamisogoddamnblind

Easy fixes

Strazyplus commented:
> Headlight overlay for drakesec and drakeeng are scuffed and backwards

fixed

CameronWoof commented on modular_skyrat/code/modules/client/loadout/__donator.dm:
```diff
@@ -21,3 +21,15 @@
+	name = "Drake Plushie"
+	category = SLOT_IN_BACKPACK
+	path = /obj/item/toy/plush/borgplush/securitydrake
+	ckeywhitelist = list("navier")
```
undocumented change

Strazyplus replied:
Added to the changelog, sorry about that.

CameronWoof commented on modular_skyrat/code/modules/client/loadout/__donator.dm:
```diff
@@ -21,3 +21,15 @@
+	name = "Drake Plushie"
+	category = SLOT_IN_BACKPACK
+	path = /obj/item/toy/plush/borgplush/doctordrake
+	ckeywhitelist = list("navier")
```
undocumented change

Strazyplus replied:
Added to the changelog, sorry about that.

Linzolle commented:
Please be aware that it is possible that the artist's [ToS](https://gdoc.pub/doc/1BtppIsljSgQ_yKVTEy1aJCkqFujjqGMY74KGCWnw6J4) are not compatible with the AGPL 3 license.

Strazyplus committed:
plushies for everyone

removes donator edits and adds plushies for backpack loadout.

Strazyplus committed:
W̱̒Ē̡̳̙̋̚͟͝ ͍̥̊̊Ã̖̝̄R̳̺͎̐̅͝E͛͜ ̩͖͗͗D̡̰͍̱͓͋͂̏̉̍Ȉ͕͍͇͚̐͌̂F͎̼̄̓F̱̿È̙̼̱̀̊R̨͌E͓̭̞͗̀̽̐ͅṊ͚͉̭̾͋͘͡T̮͔͚̊̕͠ ̛̘̼̭̽̕N̗̬͊̆́͟Ơ̘̙̏W̯̃̚͢͡ͅ.͈͖͂̚

> Rebuild of drakeborgs for downstream compatibility
> Mirrorizes code inline with citidel drakeborg PR

Gandalf2k15 reviewed (approved):
Testmerge probation complete. Final checks complete.

Iamgoofball commented on code/modules/mob/living/silicon/robot/robot_modules.dm:
```diff
@@ -1,3 +1,4 @@
+// Drakeborg sprites by Mizartz. drakeborg are licensed under the CC BY-NC-SA 3.0 license.
```
THIS IS NOT HOW LICENSES WORK

Strazyplus replied:
Absolute goofball.
